wcivf/apps/elections/templates/elections/election_view.html

```diff
@@ -41,12 +40,12 @@ <h3>{{ object.nice_election_name }}</h3>
                             {% endblocktrans %}
                         </p>
 
-                        {% if not object.in_past and not object.locked %}
+                        {% if not object.is_past and not object.locked %}
```

Comment:
I can't find the `is_past` property in the codebase, and if it's missing then this will always evaluate as True. I can only find `in_past`.